set.mm

```diff
@@ -7199,6 +7199,14 @@ _praeclarum theorema_ (splendid theorem).  (Contributed by NM,
       ( wa jca syl21anc ) ABCLDEFABCGHMIJKN $.
   $}
 
+  ${
+    syldbl2.1 $e |- ( ( ph /\ ps ) -> ( ps -> th ) ) $.
+    $( Stacked hypotheseis implies goal.  (Contributed by Stanislas Polu,
+       9-Mar-2020.) $)
+    syldbl2 $p |- ( ( ph /\ ps ) -> th ) $=
+      ( wa com12 anabsi7 ) ABCABEBCDFG $.
+  $}
+
```

Comment:
This theorem would be anabsi3, as anabsi* (5, 6, 7, 8) commuted is presumably (1, 2, 3, 4), but this is arguably not really an improvement. So just a comment, not a suggestion.

Reply:
Yeah those renaming commits should be done in a separate pr. I've just moved the theorem because mmj2 autofilled it for me. This is actually how most theorems that I move are getting moved to main. MMJ2 autofills it and I get too lazy to find a different route because it is usually the shortest.

Reply:
I think we should give theorems which are moved to main adequate and hopefully final names. That's why we have (besides other reasons) the column "New" in changes-set.txt. If we defer the renaming to future PRs, this will easily be forgotten.